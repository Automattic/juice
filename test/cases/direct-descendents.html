<html><body>
<p><em><em>woot</em></em></p>
</body></html>
