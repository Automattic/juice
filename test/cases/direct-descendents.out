<html><body>
<p><em style="background: blue; color: red;"><em style="background: blue;">woot</em></em></p>
</body></html>
